functional/ek-cert-use-ek_handle-custom-ca_certs/main.fmf

```diff
@@ -0,0 +1,34 @@
+summary:  Test of keylime checking of endorsement key by manually generate ek and verify against CA certs
+description: |
+    Running all services on localhost.
+    Uses certificates generated by keylime.
+    Manually generate endorsement key and add 
+    address of key to keylime.conf.
+    Starts verifier, registrar, agent.
+    Add empty CA certs, expected fail while adding agent.
+    Add right CA certs, pass of adding agent.
```

Comment:
Add incorrect CA certs for EK cert validation, expect agent addition to fail.
Add correct CA certs for EK cert validation, expect agent addition to pass.

Reply:
Thank you for remind me, I update the description in editor, but forgot to save it, so I overlooked it.